elastic/apm-server
# Wire kibana config from fleet

## Motivation/summary

Adds support for central config and sourcemaps in managed mode.


## Checklist



## How to test these changes

In the Kibana UI create an APM integration without (or with a wrong API key), and check that `curl http://localhost:8200/config/v1/agents\?service.name=foo` returns "Unauthorized". Then create a valid api key and update the apm policy with it. Repeat the same curl command, and this time should respond with service-does-not-exist error.

For sourcemaps: index a sourcemap, and check that APM Server can read it back when ingesting errors, provided the right API key has been configured in the policy editor.
 

## Related issues

Requires https://github.com/elastic/beats/pull/23856

Closes https://github.com/elastic/apm-server/issues/4573
